### DISCUSSION TITLE
How can I upgrade to php 8.2.23 on Laragon?

### DISCUSSION THREAD

**Discussion by BrunoFel** [leokhoa/laragon#896] [General]
How can I upgrade to php 8.2.23 on Laragon?

**Comment by ShahidJamilpk** [comment]
How to upgrade php 8.3.12

**Comment by Pen-y-Fan** [comment]
Upgrading to PHP 8.2 is straight forward, see the [Wiki: How to add another PHP version (PHP 8.3.4) ](https://github.com/leokhoa/laragon/wiki/%5BTutorial%5D-How-to-add-another-PHP-version-(PHP-8.3.4)) or my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/).

For PHP 8.3 you also need to update Apache and the VC version, see the discussion: PHP 8.3 now available, Questions on which versions are usable with Laragon Full 6.0 220916 #576 and the [Wiki: How to add another Apache version](https://github.com/leokhoa/laragon/wiki/%5BTutorial%5D-How-to-add-another-Apache-version). 

When updating Apache double check the folder structure, one common mistake, when unzipping, the Apache directory is duplicated:

- C:\laragon\bin\apache\\***apache***\httpd-....  ❌
- C:\laragon\bin\apache\httpd-.... ✅

**Comment by pitommasi** [comment]
apachelounge website is down, is there an alternative?
